src/shared/components/create-case-filters/create-case-filters.component.spec.ts

```diff
@@ -618,310 +570,26 @@ describe('CreateCaseFiltersComponent', () => {
-
-    callbackErrorsContext.trigger_text = CreateCaseFiltersComponent.TRIGGER_TEXT_CONTINUE;
-    component.callbackErrorsNotify(callbackErrorsContext);
-
-    fixture.detectChanges();
-    expect(button.nativeElement.textContent).toEqual(CreateCaseFiltersComponent.TRIGGER_TEXT_CONTINUE);
-  });
-
-  it('should clear errors and warnings', () => {
-    let callbackErrorsContext: CallbackErrorsContext = new CallbackErrorsContext();
-    callbackErrorsContext.trigger_text = CaseViewerComponent.TRIGGER_TEXT_START;
-    component.callbackErrorsNotify(callbackErrorsContext);
-
-    fixture.detectChanges();
-    component.resetErrors();
-
-    let error = de.query($ERROR_SUMMARY);
-    expect(error).toBeFalsy();
-    expect(component.error).toBeFalsy();
-    expect(component.ignoreWarning).toBeFalsy();
-  });
-
-  it('should disable button when form valid but callback validation errors exists', () => {
-    component.selected.jurisdiction = JURISDICTION_2;
-    component.selected.caseType = CASE_TYPES_2[2];
-    component.selected.event = CASE_TYPE.events[1];
-    component.error = HttpError.from({});
-    fixture.detectChanges();
-
-    let button = de.query($SUBMIT_BUTTON);
-    expect(attr(button, 'disabled')).toEqual(null);
-    const FIELD_ERRORS = [
-      {
-        x: ''
-      }
-    ];
-    const VALID_ERROR = {
-      details: {
-        field_errors: FIELD_ERRORS
-      }
-    };
-    let error = HttpError.from(VALID_ERROR);
-    component.error = error;
-    fixture.detectChanges();
-
-    expect(attr(button, 'disabled')).toEqual('');
-  });
-
-  it('should disable button when form valid but callback errors exist', () => {
-    component.selected.jurisdiction = JURISDICTION_2;
-    component.selected.caseType = CASE_TYPES_2[2];
-    component.selected.event = CASE_TYPE.events[1];
-    fixture.detectChanges();
-
-    let button = de.query($SUBMIT_BUTTON);
-
-    expect(attr(button, 'disabled')).toEqual(null);
-    let error = HttpError.from({});
-    error.callbackErrors = ['anErrors'];
-    component.error = error;
-    fixture.detectChanges();
-
-    expect(attr(button, 'disabled')).toEqual('');
-  });
-
-  it('should enable button when previously disabled for callback errors but new jurisdiction, case type and event selected', () => {
-    let error = HttpError.from({});
-    error.callbackErrors = ['anErrors'];
-    component.error = error;
-    fixture.detectChanges();
-    let button = de.query($SUBMIT_BUTTON);
-    expect(attr(button, 'disabled')).toEqual('');
+  }));
 
-    component.filterJurisdictionControl.setValue(JURISDICTION_2.id);
-    component.onJurisdictionIdChange();
-    component.selected.caseType = CASE_TYPES_2[2];
-    component.selected.event = CASE_TYPE.events[1];
-    component.error = null;
-    fixture.detectChanges();
-    expect(attr(button, 'disabled')).toEqual(null);
-    expect(mockCallbackErrorSubject.next).toHaveBeenCalledWith(null);
-    expect(mockAlertService.clear).toHaveBeenCalled();
-  });
```

Comment:
Are moving all of these tests to mgmt web now?

Reply:
As much possible, yes.

Reply:
👍